ingq/user/application/user_service.py

```
@@ -69,28 +69,18 @@ def find_user_by_email(self, email: str) -> UserVO:
-                detail="이메일 또는 비밀번호가 일치하지 않습니다.",
-            )
-
+            raise InvalidCredentialsException()
         try:
             self.crypto.verify(password, user.password)
         except Exception:
-            raise HTTPException(
-                status_code=status.HTTP_401_UNAUTHORIZED,
-                detail="이메일 또는 비밀번호가 일치하지 않습니다.",
-            )
+            raise InvalidCredentialsException()
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)